kong/core/router.lua

```diff
@@ -446,25 +446,24 @@ function _M.new(apis)
   end)
 
 
-  local grab_headers = #wildcard_hosts > 0 or next(indexes.plain_hosts) ~= nil
+  local grab_host = #wildcard_hosts > 0 or next(indexes.plain_hosts) ~= nil
```

Comment:
Ah, my fault for misreading the scope of the var. For some reason I thought it was being declared in `exec`, not `new`. Ignore me!